projects/vdk-plugins/vdk-ipython-ext/README.md

```diff
@@ -0,0 +1,53 @@
+# ipython-ext
```

Comment:
ext is not needed . We do not add "plugin" or "extension" or similar prefix/suffix to any other plugins. 

Just `vdk-ipython` .